dashboard/app/models/user_school_info.rb

```diff
@@ -13,7 +13,7 @@
 #
 
 class UserSchoolInfo < ApplicationRecord
-  validates_presence_of :user, :school_info, :start_date, :last_confirmation_date
+  validates_presence_of :user, :school_info_id, :start_date, :last_confirmation_date
```

Comment:
why is this changing from `school_info` to `school_info_id`? i don't _think_ it matters, i'm mostly curious